atdgen/bin/ag_main.ml

```diff
@@ -184,6 +185,12 @@ let main () =
           refuse to print NaN and infinities (implying -json mode
           unless another mode is specified).";
 
+    "-j-gen-modules",
+    Arg.Unit (fun () ->
+                add_generic_modules := true),
+    "
+          Add genericity-friendly modules to -j output.";
```

Comment:
Can you specify what we should expect here and/or add it to the docs?

Reply:
Added both, let me know what you think